src/ui/qgsvectorlayerpropertiesbase.ui

```diff
@@ -140,7 +140,7 @@
            <string>Labels</string>
           </property>
           <property name="toolTip">
-           <string>Labels</string>
+           <string>Enable features labeling</string>
```

Comment:
This isn't just for enabling labeling though - it's also for controlling the label properties. I think the wording should follow the same pattern as the symbology tab tooltip.

Reply:
Agreed, we should use same pattern as symbology, 'Control feature labeling'.